index.html

```diff
@@ -1,57 +1,160 @@
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
           </p>
           <h2 id="connect" class="connect-title">LET’S CONNECT</h2>
-         
+
         </div>
-        </section>
-        <section id="icons-secction">
-          <ul class="social-media-icons">
-      <li class="icons">
-        <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
-      </li>
-       <li class="icons">
-        <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>              
-      </li>
-      <li class="icons">
-        <a href="#"><img src="img/M.png" class="img-icons"></a>             
-      </li>
-      <li class="icons">
-        <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>              
-      </li>
-      <li class="icons">
-        <a href="#"><img src="img/Paz.png" class="img-icons"></a>
-      </li>
-         
-      </ul>
       </section>
+      <section id="icons-secction">
+        <ul class="social-media-icons">
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/M.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/Paz.png" class="img-icons"></a>
+          </li>
+
+        </ul>
+      </section>
+
+
+    </div>
+  </div>
+  <div class="Works">
+    <section class="screen-shot ">
+      <img src="img/Snapshoot Portfolio.png" alt="project-image" class="screenshot-img">
+    </section>
+    <section class="text-block">
+      <header class="header-3">
+        <h2 class="project-tittle">Tonic</h2>
+      </header>
+      <div>
+        <ul class="frame">
+          <li class="txt" id="canopy">CANOPY</li>
+          <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point"></li>
+          <li class="txt" id="role">Back End Dev</li>
+          <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point-2"></li>
+          <li class="txt" id="year">2015</li>
+        </ul>
+      </div>
+      <div>
+        <p id="description-2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="tags">
```

Comment:
According the previous reviewer [here](https://github.com/CarlosZ96/Portfolio/pull/2#:~:text=I%20noticed%20that,%F0%9F%91%8D), you are expected to have a total of four project cards on the website just as the template shows. I really recommend that you address this issue because as a frontend developer, you'd be given mockups and templates to build a good-looking website out of and it is an unforgivable thing to make changes to a client design that is totally different from what they had provided.

NOTE: Add three more cards to the work section before asking for another review